qa-heal.planx-pla.net/metadata/aggregate_config.json

```diff
@@ -1,1240 +1,1296 @@
 {
-    "gen3_commons": {
-        "HEAL": {
-            "mds_url": "https://qa-heal.planx-pla.net/",
-            "commons_url": "https://qa-heal.planx-pla.net/"
+  "configuration": {
+    "schema": {
+      "__manifest": {
+        "type": "array",
+        "properties": {
+          "file_name": {
+            "type": "string"
+          },
+          "file_size": {
+            "type": "integer"
+          }
+        }
+      },
+      "commons_url": {},
+      "tags": {
+        "type": "array"
+      },
+      "advSearchFilters": {
+        "type": "array",
+        "default": []
+      },
+      "study_description_summary": {},
+      "project_title": {}
+    },
+    "settings": {
+      "cache_drs": true
```

Comment:
I thought everything in aggMDS is cached, why is there a separate cache_drs field?